README.md

```diff
@@ -140,6 +140,21 @@ If you come across any problems or would like to get in touch:
 * For bug reports or feature requests,
   [have a look at the GitHub issues or create a new one](https://github.com/balena-io/balena-cli/issues/).
 
+## Deprecation policy
+
+The balena CLI uses [semver versioning](https://semver.org/), with the
+concepts of major, minor and patch version releases.
+
+The latest release of the previous major version of the balena CLI will be
+supported for one year from the date when the next major version is released.
+For example, the balena CLI v10.17.5 (the latest v10 release) will be supported for
+one year from the date when v11.0.0 is released.
```

Comment:
> The latest release of the previous major version of the balena CLI will be supported for one year [...]
>> So, as discussed in the arch call, this is all about us keeping the required endpoints working and not about bugfixing.
>> https://chat.google.com/room/AAAAgEE5JmQ/Kiptkj_j6pc

To avoid the interpretation that "supported" means bug fixing, let's be more explicit that we mean balenaCloud backend service compatibility. I suggest:

```
## Deprecation policy

Several of the balena CLI commands make use of balenaCloud backend services, such as the
balena HTTP API and the balenaCloud builder. While a specific balena CLI version may be remain
available for download and installation for a long period of time, the backend services (HTTP
API) evolve independently and may eventually become incompatible with older versions of the
balena CLI. Several commands of such older balena CLI versions would then stop working. This
deprecation policy aims at providing some clarity to balena CLI users as to the period of time
that an older balena CLI version should remain compatible with balenaCloud backend services.

This deprecation policy is expressed in terms of [semver versioning](https://semver.org/),
which is used to identify balena CLI releases with the concepts of major, minor and patch
version releases.

**The latest release of the previous major version of the balena CLI shall remain compatible
with balenaCloud backend services for one year from the date when the next major version is
released.** For example, balena CLI v10.17.5 (the latest v10 release) shall remain compatible
with balenaCloud backend services for one year from the date when v11.0.0 is released.

To be clear, this statement is limited to balenaCloud backend service compatibility and does
not extend to software maintenance such as, for example, bug fixes and security patches. Such
maintenance is normally only provided for the latest major version. For this reason, users are
advised to regularly update their installation of the balena CLI.
```

Reply:
It's a bit too descriptive imo, but I'm fine with it :+1:

Reply:
@pdcastro here is something simpler, based on your initial suggestion on the SDK:
```
The balena CLI uses [semver versioning](https://semver.org/), with the
concepts of major, minor and patch version releases.

The balenaCloud backend services required to continue supporting the operation
of latest release of the previous major version of the balena CLI, will remain
compatible for one year from the date when the next major version is released.
For example, the balenaCloud backend services which are required to support the
operation of balena CLI v10.17.5 (the latest v10 release) will remain
operational and compatible for one year from the date when v12.0.0 is released.

At the end of this support period, the older major version is considered
deprecated and some of the functionality that depends to balenaCloud backend
services might unexpectedly stop working, and you are highly encouraged to
upgrade to the to latest major version.
```

Reply:
* Instead of saying that the backend services will remain compatible with the CLI, it may be better to say that the CLI will remain compatible with the backend services. Almost the same, but the former sounds closer to "the backend will remain compatible with CLI v10.17.5", whereas the latter sounds closer to "we can release v10.17.6 so the CLI remains compatible with the backend".
* We can avoid the word "support" altogether.

New suggestion (even shorter):
```
The balena CLI uses [semver versioning](https://semver.org/), with the concepts of major,
minor and patch version releases.

The latest release of the previous major version of the balena CLI will remain compatible
with the balenaCloud backend services for one year from the date when the next major
version is released. For example, balena CLI v10.17.5, as the latest v10 release, would
remain compatible with the balenaCloud backend for one year from the date when v11.0.0 is
released.

At the end of this period, the older major version is considered deprecated and some of
the functionality that depends on balenaCloud services may stop working at any time.
Users are encouraged to regularly update the balena CLI to the latest version.
```

Reply:
Loved it @pdcastro :heart: Re-pushing in a bit :tada: